Makefile

```diff
@@ -80,10 +80,13 @@ help:  ## Display this help
 .PHONY: testprereqs
 testprereqs: $(KUBEBUILDER) $(KUSTOMIZE)
 
-.PHONY: test
-test: testprereqs generate fmt lint ## Run tests
+.PHONY: unit
+unit: testprereqs ## Run tests
```

Comment:
do you need `testprereqs` ?

Reply:
Hmm I thought we needed it for kubebuilder but looks like that is already taken care of with the scripts... But if we don't need testprereqs for the tests, what do we use it for? Is it for generate?

Reply:
Wouldnt it be needed for local tests? I am not sure if removing it is a good idea .

Reply:
No, that was removed from CAPM3 too some time ago, xref: https://github.com/metal3-io/cluster-api-provider-metal3/pull/389

Reply:
ok